pennylane/devices/device_api.py

```diff
@@ -121,6 +123,25 @@ class Device(abc.ABC):
 
     """
 
+    config: Optional[str] = None
```

Comment:
What would you think of `config_location` instead?

Reply:
How about `config_file`?